oss-internship-2020/pffft/main_pffft_sandboxed.cc

```diff
@@ -64,63 +64,72 @@ void ShowOutput(const char* name, int N, int cplx, float flops, float t0,
       printf("|      n/a   ");
   } else {
     if (flops != -1) {
-      printf("N=%5d, %s %16s : %6.0f MFlops [t=%6.0f ns, %d runs]\n", N,
+      printf("n=%5d, %s %16s : %6.0f MFlops [t=%6.0f ns, %d runs]\n", n,
              (cplx ? "CPLX" : "REAL"), name, mflops,
              (t1 - t0) / 2 / max_iter * 1e9, max_iter);
     }
   }
   fflush(stdout);
 }
 
+absl::Status PffftMain() {
+  PffftSapiSandbox sandbox;
+  SAPI_RETURN_IF_ERROR(sandbox.Init());
+
```

Comment:
I meant for you to move most of the code from your `main()` function to here, directly below line 78.